tests/filter/crlf.c

```diff
@@ -103,12 +103,12 @@ void test_filter_crlf__with_safecrlf(void)
 	cl_git_fail(git_filter_list_apply_to_data(&out, fl, &in));
 	cl_assert_equal_i(giterr_last()->klass, GITERR_FILTER);
 
-	/* Normalized \n is reversible, so does not fail with safecrlf */
+	/* Normalized \n is reversible, however, vanilla git does fail here -> see tests/index/crlf.c */
 	in.ptr = "Normal\nLF\nonly\nline-endings.\n";
 	in.size = strlen(in.ptr);
 
-	cl_git_pass(git_filter_list_apply_to_data(&out, fl, &in));
-	cl_assert_equal_s(in.ptr, out.ptr);
+	cl_git_fail(git_filter_list_apply_to_data(&out, fl, &in));
```

Comment:
I'm unsure about this change as the original comment said "Normalized \n is reversible, so does not fail with safecrlf", however, vanilla git refuses to handle this kind of problem...